src/AppMeta.tsx

```diff
@@ -1,15 +0,0 @@
-import React from 'react';
-import { Helmet } from 'react-helmet';
-import { useTranslation } from 'react-i18next';
```

Comment:
...and `react-i18next` as well?

Reply:
Has been removed.

Reply:
But it's still [there](https://github.com/City-of-Helsinki/example-ui-profile/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R43)?

Reply:
AppMeta.tsx has been deleted some time ago.

Reply:
I mean that `react-i18next` is still in `package.json` but can it be removed now?